src/Responses/ProductCategoryResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
```

Comment:
Ah yeah, I usually manually delete that but sometimes forget since phpstorm minimizes it by default. I'll go change that setting real quick